docs: Fix wording in intro tutorial (#3954)
bentoml/BentoML: docs/source/tutorial.rst

   side by side with this guide. As you go through this guide, you can simply run the
   sample code from the Colab Notebook.

>    You will be able to try out most of the tutorial content on Colab. Note that the Docker
>    container deployment section is not executable on Google Colab due to its current lack of
>    Docker support.

Reviewer: ```suggestion
   You will be able to try out most of the tutorial content on Colab. Note that deploying as Docker containers on Google Colab is not supported due to Colab's lack thereof.
```

Author: Changed. Thanks!
